src/app.rs

```diff
@@ -37,6 +38,27 @@ impl ToResponseCode for InclusionProofResponse {
     }
 }
 
+#[derive(Serialize)]
+pub enum VerifyProofResponse {
+    Ok,
```

Comment:
I think it would be better to just let the clients decide root expiration. I.e. provide them with root timestamp, rather than `Ok` vs `Expired`.